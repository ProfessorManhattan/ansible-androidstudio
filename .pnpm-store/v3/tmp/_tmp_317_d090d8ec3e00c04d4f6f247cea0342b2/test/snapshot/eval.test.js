const parser = require("../main");

describe("eval", function () {
  it("simple", function () {
    expect(parser.parseEval('eval("command");')).toMatchSnapshot();
  });
});
