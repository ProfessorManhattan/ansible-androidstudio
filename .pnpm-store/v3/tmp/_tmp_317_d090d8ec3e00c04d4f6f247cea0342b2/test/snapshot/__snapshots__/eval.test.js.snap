// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`eval simple 1`] = `
Program {
  "children": Array [
    ExpressionStatement {
      "expression": Eval {
        "kind": "eval",
        "source": String {
          "isDoubleQuote": true,
          "kind": "string",
          "raw": "\\"command\\"",
          "unicode": false,
          "value": "command",
        },
      },
      "kind": "expressionstatement",
    },
  ],
  "errors": Array [],
  "kind": "program",
}
`;
